pipeline/data/parallel_importer.py

```diff
@@ -25,6 +25,7 @@
 from opustrainer.modifiers.typos import TypoModifier
 from opustrainer.types import Modifier
 
+from pipeline.alignments.tokenizer import IcuTokenizer
```

Comment:
The dependencies from here have been included in eval.txt, but I'm not sure if it's the way to do it or the kind.yml should install both alignments and eval dependencies.

Reply:
The kind for evals should install only evals dependencies